src/js/coblocks-google-maps.js

```diff
@@ -1,1064 +1,1057 @@
 'use strict';
 
-import jQuery from 'jquery';
-
 /*global google coblocksGoogleMaps*/
 
-const coblocksMap = {
-	init() {
-		const self = this;
-		self.elem = jQuery( '.wp-block-coblocks-map' );
-
-		self.renderMap();
-	},
-
-	renderMap() {
-		const self = this;
-		self.elem.each( function( index, Element ) {
-			let mapAttributes = jQuery( this ).attr( 'data-map-attr' );
-			mapAttributes = mapAttributes.replace( /\/q/g, '"' ).split( '||' );
-			const gmapAttr = JSON.parse( '{' + mapAttributes + '}' );
-
-			if (
-				gmapAttr.address === 'undefined' &&
-				gmapAttr.lat === 'undefined' &&
-				gmapAttr.lng === 'undefined'
-			) {
-				return;
-			}
-
-			let skin = self.mapStyles( gmapAttr.skin );
-			if ( typeof skin === 'undefined' ) {
-				skin = [];
-			}
-			skin.push(
-				{
-					featureType: 'administrative.land_parcel',
-					elementType: 'labels',
-					stylers: [
-						{
-							visibility: 'off',
-						},
-					],
-				},
-				{
-					featureType: 'poi',
-					elementType: 'labels.text',
-					stylers: [
-						{
-							visibility: 'off',
-						},
-					],
-				},
-				{
-					featureType: 'poi.business',
-					stylers: [
-						{
-							visibility: 'off',
-						},
-					],
-				},
-				{
-					featureType: 'transit',
-					stylers: [
-						{
-							visibility: 'off',
-						},
-					],
-				}
-			);
-
-			const map = new google.maps.Map( Element, {
-				zoom: parseInt( gmapAttr.zoom ),
-				center: new google.maps.LatLng( gmapAttr.lat, gmapAttr.lng ),
-				styles: skin,
-				mapTypeControl: gmapAttr.mapTypeControl === 'true' ? true : false,
-				zoomControl: gmapAttr.zoomControl === 'true' ? true : false,
-				streetViewControl: gmapAttr.streetViewControl === 'true' ? true : false,
-				fullscreenControl: gmapAttr.fullscreenControl === 'true' ? true : false,
-			} );
+const renderMap = ( elements ) => {
+	Array.from( elements ).forEach( ( element ) => {
+		let mapAttributes = element.getAttribute( 'data-map-attr' );
+		mapAttributes = mapAttributes.replace( /\/q/g, '"' ).split( '||' );
+		const gmapAttr = JSON.parse( '{' + mapAttributes + '}' );
 
-			new google.maps.Marker( {
-				position: new google.maps.LatLng( gmapAttr.lat, gmapAttr.lng ),
-				map,
-				icon: {
-					url: coblocksGoogleMaps.url + '/dist/images/markers/' + gmapAttr.skin + '.svg',
-					scaledSize: new google.maps.Size(
-						gmapAttr.iconSize,
-						gmapAttr.iconSize
-					),
-				},
-			} );
-			map.setCenter( new google.maps.LatLng( gmapAttr.lat, gmapAttr.lng ) );
-		} );
-	},
-
-	mapStyles( key ) {
-		const GMapStyles = {};
+		if (
```

Comment:
I wonder if it would not be cleaner to also refactor this like
```js
if ( [ gmapAttr.address, gmapAttr.lat, gmapAttr.lng ].includes('undefined') ) { 
    return;
}
```